src/test/unit_tests/test_system_servers.js

```diff
@@ -193,6 +193,33 @@ mocha.describe('system_servers', function() {
         await rpc_client.events.read_activity_log({ limit: 2016 });
     });
 
+    mocha.it('Calculate md5_etag for account works', async function() {
+        await rpc_client.account.create_account({
+            name: EMAIL1,
+            email: EMAIL1,
+            has_login: false,
+            s3_access: true,
+            force_md5_etag: true,
+        });
+        let info = await rpc_client.account.read_account({
+            email: EMAIL1,
+        });
+        assert(info.force_md5_etag === true);
+
```

Comment:
Maybe we can also test if after setting this true NSFS really enforces MD5 calculation? Will be helpful to avoid regressions when someone else is working on this area.

Reply:
Thanks, yes your are right.

We are not testing the complete flow here, this test is only to check the account part. I think, we can have one more test to check the whole flow of md5 calculation but for this PR the above test should be fine. @dannyzaken wdyt??

Reply:
in the system servers test we mainly tests the APIs. in this case the force_md5 flag is also not having much effect, since it is only used for NSFS flows.
I agree that we should have separate tests for the functionality of the md5 calculations in NSFS